syntax = "proto3";

import "hq/dto/Security.proto";
import "hq/dto/BasicQot.proto";
option java_package = "com.huasheng.quant.open.sdk.protobuf.hq.notify";
option java_outer_classname = "BasicQotNotifyProto";
message BasicQotNotify {
    Security security = 1;
    BasicQot basicQot=2; // 基础行情返回
}